chain/chaindb.go

```diff
@@ -552,7 +531,7 @@ func (cdb *ChainDB) dropBlock(dropNo types.BlockNo) error {
 	dbTx.Delete(dropIdx)
 
 	// update latest
-	dbTx.Set(latestKey, newLatestIdx)
+	dbTx.Set([]byte(schema.Latest), newLatestIdx)
```

Comment:
The use of `[]byte(` on every `Get` and `Set` is annoying
The keys could be stored already in `[]byte` format

Reply:
it is because []byte can`t be declared by const. if it declared just var, it can changed on other packages.

Reply:
I think declare string literal to const is safer than than declaring all keys as var.

Reply:
Using `[]byte()` everywhere is awful. It was not like that before. It may have a way to fix it

Reply:
All these keys were `var` already, so no reason to make the usage more ugly.
Option 1: Continue using `var` for the keys
Option 2: Change the `Set` and `SetData` to accept string for the key

Reply:
I like the idea of using functions to return the key:
* Solves the problem of collision of key name and function name
* Makes the code shorter without the need of `[]byte()`
* Creates a "standard": use function for keys both when there is and there is no argument

The keys can be stored as const string and with the word `Key` at the end of variable name and the functions without `Key` because there is already on the `dbkey` package

Example:
```go
const {
  ...
  LatestBlockKey = ChainDBName + ".latest"
  ...
}

...

func LatestBlock() []byte {
  return []byte(LatestBlockKey)
}
```

Reply:
And if the keys will not be accessed directly, they can be declared as private
Example: `latestBlockKey`

Reply:
good idea. i`ll change as you said.